index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+    <h1 id="title">Survey Form</h1>
+    <p id="description"> Survey carried out in 2020</p><br>
+    <form id="survey-form">
+        <input list="dropdown">
+        <datalist id="dropdown">
+            <option value="1-4">
+            <option value="1-4">
+            <option value="1-4">
+        </datalist><br><br>
```

Comment:
Let's add an appropriate `label` to this `input`. 

A suggested label would be the month of the year the survey is taken.

Also, note that the `option` element can also have both `label` and `value`, this can be done by wrapping the `option` element with the `label`. e.g.

```html
<option value="1">January</option>
```